news/templatetags/news.py

```diff
@@ -20,25 +20,23 @@ def user_news(context, num, more=0):
     news = context['course_news']
 
     if context['is_course_staff']:
-        alerts,news = news.for_staff()
+        news = news.for_staff()
     else:
         user = context['request'].user
-        alerts,news = news.for_user(
+        news = news.for_user(
```

Comment:
> Therei s a change from (a, b) to (a), i.e., from tuple to a single item. If this is supposed to return only a single value and not a tuple, then you should remove the brackets.

As the data is read corectly here, you should remove brackets from the cached/*.py file.

Reply:
Aaand done: 
https://github.com/apluslms/a-plus/pull/570/commits/458e4b11841b3b55a1042dedba903d6dd22508f3